Cython/Debugger/Cygdb.py

```diff
@@ -79,8 +85,8 @@ def make_command_file(path_to_debug_info, prefix_code='', no_import=False):
                     gdb.lookup_type('PyModuleObject')
                 except RuntimeError:
                     sys.stderr.write(
-                        'Python was not compiled with debug symbols (or it was '
-                        'stripped). Some functionality may not work (properly).\\n')
+                        "''' + interpreter + ''' was not compiled with debug symbols (or it was "
```

Comment:
Someone commented "Did you actually try this? I don't think it inserts the interpreter name here.", but I can't find find the comment.
Yes I tried this. interpreter is the variable defined by interpreter = interpreter_file.read() . Note that this is not the current python interpreter (that would be sys.executable) but the interpreter that compiled the users .pyx files.
Example:
```
$ python2 setup.py build_ext --inplace   
...
$ cygdb . -- -q --args python main.py
Reading symbols from python...
(No debugging symbols found in python)
/usr/bin/python2 was not compiled with debug symbols (or it was stripped). Some functionality may not work (properly).
(gdb)
```

Reply:
Yeah, my bad. I failed to understand the string quoting in the diff. All fine.